[WIP]README database design

MoritaDaichi opened the pull request:
データベースの設計図を記述しました

Commit:
README database design

データベースの設計図を記述しました

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+
+### Asosiation
+- has_many :members
+- belongs_to :email
```
emailはdeviseでusersテーブルに入っているはずですね

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+|name|string|null: false|
+
+### Asosiation
+- has_many :members
```
アソシエーションを考えましょう

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|user_id|integer|primary_key :true|
```
これはいりますか？

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+- has_many :members
+- belongs_to :email
+
+## emailテーブル
```
これはいらないですね

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+- belongs_to :user
+- has_many :messages
+
+##  groupテーブル
```
複数形にしましょう

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+|title|string|null: false|
+
+### Asosiation
+- has_many :members
```
これだけで足りますか？

b-i-hide commented on README.md:
```diff
@@ -13,6 +13,62 @@ Things you may want to cover:
+|image|string||
+
+### Asosiation
+- has_many :members
```
再考しましょう。

b-i-hide commented:
全体的にできていないので、もう一度しっかりと考えてください。

Commit:
設計修正

指摘された箇所を修正しました。

Commit:
usersテーブルのカラムの項目を削除

yuki-ar commented on README.md:
```diff
@@ -13,6 +13,49 @@ Things you may want to cover:
+
+### Asosiation
+- belongs_to member
+- has_many groups
```
messageは複数のグループに属しますか？

yuki-ar commented on README.md:
```diff
@@ -13,6 +13,49 @@ Things you may want to cover:
+## usersテーブル
+
+### Asosiation
+- has_many :members
```
中間テーブルを介して何と関係があるかがわからないです。
中間テーブルのカリキュラムを読んで見て下さい。

yuki-ar commented on README.md:
```diff
@@ -13,6 +13,49 @@ Things you may want to cover:
+|image|string|null: true|
+
+### Asosiation
+- belongs_to member
```
memberとは何でしょうか？

yuki-ar commented on README.md:
```diff
@@ -13,6 +13,49 @@ Things you may want to cover:
+### Asosiation
+- belongs_to :group
+- belongs_to :user
+- has_many :messages
```
中間テーブルのカリキュラムを読んで見て下さい。

Commit:
2回目修正

指摘された箇所を修正しました

Commit:
garbage commit

操作ミスで別のブランチで作業していたファイルです。ブランチをコミットしないと切り替えることができなかったのでコミットします。

Commit:
delete members table

membersテーブルをmessagesテーブルと結合させ、usersテーブルのuser_idにprimary_keyを設定し、groupsテーブルのgroup_idにprimary_keyを設定しました。

yuki-uchida commented on README.md:
```diff
@@ -13,6 +13,40 @@ Things you may want to cover:
+|------|----|-------|
+|group_id|integer|primary_key: true|
+|title|string|null: false|
+
```
とあるグループに、どのユーザーが所属しているかをどうやって判別すれば良いかをもう一度考えてみましょう！

Commit:
add user_id in groupsテーブル

groupsテーブルにuser_idを追加し、それに付随するアソシエーションをusersテーブルとgroupsテーブルを追加しました。

yuki-ar reviewed (commented):
read.mdとindexが混ざっているので分けましょう👁

MoritaDaichi committed:
Delete index.scss
